config.toml

```diff
@@ -22,6 +22,12 @@ buildFuture = true # A new event website will always be in the future
   gcse = "011805775785170369411:p3ec0igo0qq"
   js = "js/solstice.js?v1"
   featured_content_publish_target = "eclipse_org"
+  header_left_classes = "col-sm-8 col-xs-8 col-md-12"
+  header_right_classes = "col-sm-8 col-xs-8 col-md-12"
```

Comment:
This is good on boh mobile and desktop view, you can change here, but in order to make the mobile view looks good, I think col-xs-8 is good.